conf/messages.en

```diff
@@ -524,7 +524,7 @@ cds.matching.organisation-type.user.header=What do you want to apply as?
 cds.matching.organisation-type.organisation-type=Business type
 cds.matching.organisation-type.radio.company.label=Company
 cds.matching.organisation-type.radio.organisation.label=Organisation
-cds.matching.organisation-type.radio.organisation.hint-text=For example a limited company, corporation, limited liability company or charity
+cds.matching.organisation-type.radio.organisation.hint-text=For example, a limited company, corporation, limited liability company or charity
```

Comment:
Are you sure we should add a comma here? Worth checking with Colin...Also the Welsh of course.

Reply:
I checked with Colin, the comma should be there. The welsh is correct

Reply:
👍